charts/internal/cilium/charts/config/templates/label-prefix-config.yaml

```diff
@@ -0,0 +1,78 @@
+# Source: https://docs.cilium.io/en/stable/operations/performance/scalability/identity-relevant-labels/#identity-relevant-labels
```

Comment:
Can you please change the order so that it resembles the source? Then it is probably easier to keep it up-to-date.